crates/humanode-runtime/src/tests/fixed_supply.rs

```diff
@@ -160,106 +182,32 @@ fn total_issuance_transaction_payment_validate() {
 }
 
 #[test]
-fn total_issuance_evm_withdraw() {
-    // Build the state from the config.
-    new_test_ext_with().execute_with(move || {
-        let existential_deposit =
-            <<Runtime as pallet_balances::Config>::ExistentialDeposit as Get<u128>>::get();
-
-        // Check total issuance before making evm withdraw.
-        let total_issuance_before = Balances::total_issuance();
-
-        let bob_evm = evm_truncated_address(account_id("Bob"));
-        let hashed_bob_evm =
-            <Runtime as pallet_evm::Config>::AddressMapping::into_account_id(bob_evm);
-
-        // Send tokens to hashed_bob_evm to make withdraw from bob_evm.
-        assert_ok!(Balances::transfer(
-            Some(account_id("Bob")).into(),
-            hashed_bob_evm.into(),
-            INIT_BALANCE - existential_deposit - 1,
-        ));
-
-        assert_ok!(EVM::withdraw(Some(account_id("Bob")).into(), bob_evm, 1000));
-
-        // Check total issuance after making evm withdraw.
-        assert_eq!(Balances::total_issuance(), total_issuance_before);
-    })
-}
-
-#[test]
-fn total_issuance_evm_call() {
-    // Build the state from the config.
-    new_test_ext_with().execute_with(move || {
-        let existential_deposit =
-            <<Runtime as pallet_balances::Config>::ExistentialDeposit as Get<u128>>::get();
-
-        // Check total issuance before making evm call.
-        let total_issuance_before = Balances::total_issuance();
-
-        let bob_evm = evm_truncated_address(account_id("Bob"));
-        let hashed_bob_evm =
-            <Runtime as pallet_evm::Config>::AddressMapping::into_account_id(bob_evm);
-
-        // Send tokens to hashed_bob_evm to make call from bob_evm.
-        assert_ok!(Balances::transfer(
-            Some(account_id("Bob")).into(),
-            hashed_bob_evm.into(),
-            INIT_BALANCE - existential_deposit - 1,
-        ));
-
-        assert_ok!(EVM::call(
-            Some(account_id("Bob")).into(),
-            bob_evm,
-            H160::from_str("1000000000000000000000000000000000000001").unwrap(),
-            Vec::new(),
-            U256::from(1_000),
-            1000000,
-            U256::from(2_000_000_000),
-            Some(U256::from(1)),
-            Some(U256::from(0)),
-            Vec::new(),
-        ));
-
-        // Check total issuance after making evm call.
-        assert_eq!(Balances::total_issuance(), total_issuance_before);
-    })
-}
-
-#[test]
-fn total_issuance_evm_create() {
```

Comment:
We should still test that transaction and execution don't alter the total issuance

Reply:
Returned - https://github.com/humanode-network/humanode/pull/673/commits/7a02d83e419c85ac87226522fb51645781454024